tapgarden/planter.go

```diff
@@ -1133,6 +1133,40 @@ func (c *ChainPlanter) prepAssetSeedling(ctx context.Context,
 		}
 	}
 
+	if c.pendingBatch != nil {
+		if _, ok := c.pendingBatch.Seedlings[req.AssetName]; ok {
```

Comment:
Thought we already had this check elsewhere?

Reply:
Yes, right now it's checked in `batch.addSeedling` - I added it here so it would happen earlier before we derive the script key for the seedling, but forgot to remove `addSeedling`.